cookbook/security/remember_me.rst

```diff
@@ -175,21 +175,25 @@ In the following example, the action is only allowed if the user has the
     The ``security.authorization_checker`` service was introduced in Symfony 2.6. Prior
     to Symfony 2.6, you had to use the ``isGranted()`` method of the ``security.context`` service.
 
-You can also choose to install and use the optional JMSSecurityExtraBundle_,
-which can secure your controller using annotations:
+If your application is based on the Symfony Standard Edition, you can also secure
+your controller using annotations:
 
 .. code-block:: php
 
-    use JMS\SecurityExtraBundle\Annotation\Secure;
+    use Sensio\Bundle\FrameworkExtraBundle\Configuration\Security;
 
     /**
-     * @Secure(roles="IS_AUTHENTICATED_FULLY")
+     * @Security("has_role('IS_AUTHENTICATED_FULLY')")
      */
     public function editAction($name)
     {
         // ...
     }
 
+.. versionadded:: 2.4
```

Comment:
as 2.4 was deprecated when 2.6 was released, we do not show any 2.4 versionadded in the 2.6 docs.